src/components/ApplicationDetails/WorkspaceSwitcher.tsx

```diff
@@ -60,11 +25,12 @@ export const WorkspaceSwitcher: React.FC<{ selectedWorkspace?: string }> = ({
       selectedItem={{ key: selectedItem.metadata.name, name: selectedItem.metadata.name }}
       onSelect={onSelect}
       footer={
-        <Level>
-          <LevelItem>
-            <Link to="#">View workspaces list</Link>
-          </LevelItem>
-        </Level>
+        <></>
```

Comment:
Don't pass the footer prop. It is creating unnecessary white space at the bottom. We're not sure when we'll have the workspace list page anyways.